[CIE-5504] c8yedge config docs (#4146)
SoftwareAG/c8y-docs: content/change-logs/edge/edge-2025.0.12-c8yedge-config.md

---
date: 2024-01-13
title: c8yedge supports configuration
change_type:
  - value: change-2c7RdTdXo4
    label: Feature
product_area: Edge
component:
  - value: component-g7hnfbu4J
    label: Edge
build_artifact:
  - value: tc-nJH2U7g3u
    label: edge
ticket: CIE-5481
version: 2025.0.12
---

> The **c8yedge** tool now supports post-installation configurations. You can use the tool, to configure a custom domain, license key, SSL certificate, register Edge in the cloud tenant etc.

Reviewer: ```suggestion
The **c8yedge** tool now supports post-installation configurations. You can use the tool to configure a custom domain, license key, and SSL certificate, to register Edge in the cloud tenant, and more.
```

Author: @BeateRixen , your suggesting gives an impression that we are asking user to "configure SSL certificate to register Edge in the cloud tenant", actually all 4 (domain, license, SSL cert and register Edge in the cloud) are independent configurations.

I intend something like below:

The **c8yedge** tool now supports post-installation configurations. You can use the tool to configure a custom domain, license key, SSL certificate, connect Edge to cloud (registering Edge in the cloud tenant), and more.

Author: Or we can simplify by removing the "register Edge in the cloud tenant" from the statement.

The **c8yedge** tool now supports post-installation configurations. You can use the tool to configure a custom domain, license key, SSL certificate, and more.

Please suggest.

Reviewer 2: I think it's the 'and' and the 'to' that's a problem here, implying that "SSL certificate" and registration on cloud are special.

I appreciate using "and more" in place of the Latin-centric " etc., but I think the homogenous comma-separated list before that gives the correct impression of this being merely a bag of settings.

Author: Addressed in https://github.com/Cumulocity-IoT/c8y-docs/pull/4146/commits/446338b400d71ae4aca810ccd2bb377f0e373169

> The **c8yedge** tool now supports post-installation configurations. You can use the tool, to configure a custom domain, license key, SSL certificate, register Edge in the cloud tenant etc.

Reviewer 2: ```suggestion
The **c8yedge** tool now supports post-installation configurations. You can use the tool to configure a custom domain, license key, SSL certificate, register Edge in the cloud tenant etc.
```

> The **c8yedge** tool now supports post-installation configurations. You can use the tool, to configure a custom domain, license key, SSL certificate, register Edge in the cloud tenant etc.

Reviewer 2: ```suggestion
The **c8yedge** tool now supports post-installation configuration. You can use the tool, to configure a custom domain, license key, SSL certificate, register Edge in the cloud tenant etc.
```
"configurations" implies an enumeration of different setups, "configuration" is the verb.

Author: Addressed in https://github.com/Cumulocity-IoT/c8y-docs/pull/4146/commits/446338b400d71ae4aca810ccd2bb377f0e373169

Reviewer 2: Think you resolved the wrong conversation - this one was a single-character grammatical point :-D

Author: Got it. My oversight. Its now addressed in https://github.com/Cumulocity-IoT/c8y-docs/pull/4146/commits/062e8bc3222c189af95542eb232f4debce9ad87e
